db/backend_test.go

```diff
@@ -2,42 +2,84 @@ package db
 
 import (
 	"fmt"
+	"os"
+	"path"
 	"testing"
 
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 	cmn "github.com/tendermint/tmlibs/common"
 )
 
-func testBackend(t *testing.T, backend string) {
+func cleanupDBDir(dir, name string) {
+	os.RemoveAll(path.Join(dir, name) + ".db")
```

Comment:
`filepath` please (path does not work on Windows)

Reply:
:+1: